### PR TITLE
Bump deps

### PR DESCRIPTION
**Pull request by AlekSi**
# Description

To stay current.

## Readiness checklist

- [ ] I added/updated unit tests.
- [ ] I added/updated integration/compatibility tests.
- [ ] I added/updated comments and checked rendering.
- [ ] I made spot refactorings.
- [ ] I updated user documentation.
- [x] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

**Review by chilagrow — APPROVED** [review]
I like the new stricter linter rules for importing :hugs:

File: tools/go.mod [hunk 1]
```diff
@@ -4,15 +4,15 @@ go 1.20
 
 require (
 	github.com/BurntSushi/go-sumtype v0.0.0-20221020234012-480526a59796
-	github.com/go-task/task/v3 v3.25.0
+	github.com/go-task/task/v3 v3.20.0
```
[the remark this answers is not in the document]
Comment (AlekSi) [reply tools/go.mod]: Nice catch! There is some bug that always downgrades that tool

**Review by chilagrow — APPROVED** [review]

File: tools/go.mod (hunk 1 above)
[the remark this answers is not in the document]
Comment (chilagrow) [reply tools/go.mod]: I see I see. Looks good now!

**Review by rumyantseva — APPROVED** [review]
LGTM

**Pull request merged**